src/view/cards/Certification/__tests__/Certification.spec.tsx

```diff
@@ -0,0 +1,36 @@
+import userEvent from '@testing-library/user-event';
+import React from 'react';
+import { render, screen } from 'utils/testing';
+import { mockCerts } from 'services/api/__mocks__';
+import { CertificationCard } from '../Certification';
+
+describe('<Certificates />', () => {
+  const { data: CertificateData } = mockCerts;
+  it('renders', () => {
+    const { container } = render(
+      <CertificationCard certification={CertificateData[0]} />,
+    );
+    expect(container).toMatchSnapshot();
+  });
```

Comment:
Having a newline after each `it()` makes things a bit more readable imo.

```suggestion
  });
  
```